common/src/main/java/com/wynntils/screens/maps/MainMapScreen.java

```diff
@@ -99,6 +99,28 @@ protected void doInit() {
                                 .withStyle(ChatFormatting.GRAY)
                                 .append(Component.translatable("screens.wynntils.map.help.description9")))));
 
+        this.addRenderableWidget(new BasicTexturedButton(
+                width / 2 - Texture.MAP_BUTTONS_BACKGROUND.width() / 2 + 6 + 20 * 3,
+                (int) (this.renderHeight
+                        - this.renderedBorderYOffset
+                        - Texture.MAP_BUTTONS_BACKGROUND.height() / 2
+                        - 6),
+                16,
+                16,
+                Texture.MAP_MANAGER_BUTTON,
+                (b) -> McUtils.mc().setScreen(PoiManagementScreen.create(this)),
+                List.of(
+                        Component.literal("[>] ")
+                                .withStyle(ChatFormatting.RED)
+                                .append(Component.translatable("screens.wynntils.map.manager.name")),
+                        Component.translatable("screens.wynntils.map.manager.description1")
```

Comment:
Did you intend for the descriptions to come on separate lines? Then you have to send them as separate item in the list. By using append they will all become a single line -- this may be fine, but if that is the intention you can have just a single i18n key and put it all on a single line.

Reply:
While this sounds logical, it might not be the case, I am pretty sure this is a copy of what I did above, and it works.

Reply:
Okay but then you don't need three different i18n keys, you can just have a single `description` key.